docs/dev/contribute.rst

```diff
@@ -87,12 +87,15 @@ projects, feel free to suggest the language on Transifex to start the process.
 
 .. _our projects on Transifex: https://www.transifex.com/projects/p/readthedocs/
 
-Triaging tickets
-----------------
+.. _contribute_triage:
 
-Core team uses the following guidelines for issue triage on all of our projects.
-These guidelines describe the issue lifecycle, and what is required at each step
-of this lifecycle.
+Triaging issues
+---------------
+
+Everyone is encouraged to help improving, refining, verifying and prioritizing
+issues on Github. The Read the Docs core :doc:`team` uses the following
```

Comment:
We prefer to use absolute paths (in case pages are moved)

```suggestion
issues on Github. The Read the Docs core :doc:`/team` uses the following
```

Reply:
``:doc:`/team` `` breaks correctly (because there is no such document)

There is actually no document `/team` and Intersphinx was automatically converting ``:doc:`team` `` to https://www.writethedocs.org/team/ -- I copy-pasted without looking at where it was pointing. I'll remove this reference from the original `note::`.

It seems like a good idea to use absolute paths. In case `/dev/teams.rst` has existed (I don't see it in the git repo history though), removing the file would also break the build, which is what we want.

Reply:
We should probably use this config https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_disabled_reftypes to avoid this happening again.